### ISSUE TITLE
This app is unable to link additional users because it has been frozen pending review.

### ISSUE THREAD

**Issue by MoAl78741** [stephanie-wang/kindlebox#14]
Getting the error..
This app is unable to link additional users because it has been frozen pending review.
--------------------------------------------------------------------------------------------
https://www.dropboxforum.com/t5/API-support/App-has-been-frozen-pending-review/td-p/171193
You'll need to apply for production to enable more users to link to your app. You can find more information on this here:
https://www.dropbox.com/developers/reference/devguide#production-approval

**Comment by Oldenborg** [comment]
I have this issue as well

**Comment by LeoFidjeland** [comment]
so do I

**Comment by sultanmeghji** [comment]
As do I... any updates?

**Comment by Oldenborg** [comment]
Nah. I have been watching this for a while.. I think the app is just dead.

**Comment by petehobo** [comment]
Adding my +1 more in hope than expectation...

**Comment by mustafa786786** [comment]
Dropbox will only allow 50 users for app registration, after that we need to pay.
If we are after 50, it will display this message.
Solution is to extend the user count in Drop box

**Comment by RiabininDmitriy** [comment]
https://www.dropbox.com/developers/reference/developer-guide#production-approval
https://www.dropboxforum.com/t5/API-Support-Feedback/App-has-been-frozen-pending-review/m-p/171194#M17118
